gammapy/visualization/utils.py

```diff
@@ -204,9 +205,10 @@ def plot_distribution(
         Axis object to plot on. If a list of Axis is provided it has to be the same length as the length of _map.data.
     ncols : int
         Number of columns to plot if a "plot grid" was to be done.
-    func : function object
-        The function to pass to `scipy.optimize.curve_fit`. Default is None.
-        If None, no fit will be done.
+    func : function object or str
+        The function to used to fit a map data histogram or "norm". Default is None.
```

Comment:
The wording here does not make sense
`The function is used`?

Reply:
I actually think `The function used to ...` is better, what do you think ?